### ISSUE TITLE
No documentation for the rem() mixin

### ISSUE THREAD

**Issue by joseluis** [Compass/compass#1915]
On Page: http://compass-style.org/index/mixins/

I was looking for:
the rem() mixin, which is defined in file stylesheets/compass/typography/_units.scss

But instead I found:
documentation can't be found

**Comment by dcorb** [comment]
Agree. This is quite useful, should be documented. Or is it only for internal use ?

**Comment by dcorb** [comment]
The rem mixin was added in this PR https://github.com/Compass/compass/pull/896 and documentation was forgotten
